birdhouse/env.local.example

```diff
@@ -398,6 +398,27 @@ export GEOSERVER_ADMIN_PASSWORD=geoserverpass
 # Magpie DB name
 #export MAGPIE_DB_NAME="magpiedb"
 
+# Magpie user registration and approvals
```

Comment:
references to the following would be useful here: 
- https://pavics-magpie.readthedocs.io/en/latest/configuration.html#user-registration-and-approval-configuration
- https://pavics-magpie.readthedocs.io/en/latest/configuration.html#application-settings
- https://pavics-magpie.readthedocs.io/en/latest/authentication.html#user-registration